conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node

```diff
@@ -22,5 +22,6 @@ server_node:
     site_id: TMPL_SITE_ID
     rack_id: TMPL_RACK_ID
     storage_set_id: TMPL_STORAGE_SET_ID
+    host_id: TMPL_HOST_ID
```

Comment:
what is this key? Why is it required? There is nothing like host_id in mini-provisioner confstore - https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys

Reply:
Yeah good catch, it has to be "server_node>{machine_id}>network>management>public_fqdn

Reply:
are we using it in mini provision ha ?

if yes 
please create pr to update dev_env for main branch
https://github.com/Seagate/cortx-ha/blob/main/jenkins/dev_env/conf/example_config.json

Reply:
@ajay-paratmandali As i changed "host_id" to "public_fqdn", now there is no need to update dev_env .
"public_fqdn" is already present there.